lib/begin.go

```diff
@@ -179,6 +189,9 @@ func (a *ACBuild) beginFromImage(start string, insecure bool) error {
 
 	err = reg.Fetch(app.Name, labels, 0, false)
 	if err != nil {
+		if strings.Contains(err.Error(), "no such host") {
```

Comment:
I guess checking the string isn't ideal. Just pushed a commit, I now do a type assertion on err to see if it's a `*url.Error`, and if it's ok return the helpful error.